### PR TITLE
CORTX-29555: Cleanup/Archive stale code in CSM

### PR DESCRIPTION
**Pull request by rohitkolapkar**
Signed-off-by: Pranali Ugale <pranali.ugale@seagate.com>
Signed-off-by: Rohit Kolapkar <rohit.j.kolapkar@seagate.com>

# Problem Statement
- [F-66C: Archive unused CSM features](https://jts.seagate.com/browse/CORTX-29554)
- [F-66C: Remove stale/obsolete features from CSM](https://jts.seagate.com/browse/CORTX-29555)

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

# Dev Tessting
1. CSM User Management test cases: Passed
image

2. IAM users APIs
Create IAM user
image

GET IAM user
image

Modify IAM user
image

Delete IAM User
image

3. Support Bundle generation- success
[support_bundle.txt](https://github.com/Seagate/cortx-manager/files/8388766/support_bundle.txt)

4. Pods status
image

### CODE REVIEW

**Comment by rohitkolapkar** [comment]
Custom ci : success
cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5508-custom-ci
cortx-docker.colo.seagate.com/seagate/cortx-rgw:2.0.0-5508-custom-ci

3N node: success
https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/RGW/job/setup-cortx-rgw-cluster/1921/

Pods status:
image

**Review by Pranali04796 — APPROVED** [review]

**Review by abhijit1patil — APPROVED** [review]

File: csm/cli/support_bundle/__init__.py [hunk 1]
```diff
@@ -12,7 +12,3 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
```
Comment (avrnv) [inline csm/cli/support_bundle/__init__.py:14]: Minor: add the new line at the end of the file, please.

File: csm/core/agent/api.py [hunk 2]
```diff
@@ -560,7 +529,9 @@ async def process_websocket(request):
     async def _on_startup(app):
         Log.debug('REST API startup')
         CsmRestApi._bgtasks.append(app.loop.create_task(CsmRestApi._websock_bg()))
-        CsmRestApi._bgtasks.append(app.loop.create_task(CsmRestApi._ssl_cert_check_bg()))
+
+        # Commented as securityservice and provisioner plugin is archieved.
+        # CsmRestApi._bgtasks.append(app.loop.create_task(CsmRestApi._ssl_cert_check_bg()))
```
Comment (avrnv) [inline csm/core/agent/api.py:534]: Minor: "~~is archieved~~ are archived". why is this comment needed? maybe, just remove these 2 lines?

**Review by avrnv — APPROVED** [review]

File: csm/core/agent/api.py (hunk 2 above)
Comment (rohitkolapkar) [reply csm/core/agent/api.py:535]: updated the comment.

File: csm/cli/support_bundle/__init__.py (hunk 1 above)
Comment (rohitkolapkar) [reply csm/cli/support_bundle/__init__.py:14]: updated.

**Review by pawankumarsrivastava — APPROVED** [review]

**Comment by rohitkolapkar** [comment]
retest this please

**Pull request merged**